lib/widgets/home.dart

```diff
@@ -148,40 +148,6 @@ class _HomePageState extends State<HomePage> {
   }
```

Comment:
> page: Move no-content placeholder widget to page.dart, from home.dart 
 
 Should this commit be marked as nfc?